plugins/modules/vmware_object_role_permission_info.py

```diff
@@ -26,6 +26,12 @@
     - "python >= 2.7"
     - PyVmomi
 options:
+  principal:
```

Comment:
Please create a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment). See [this fragment](https://github.com/ansible/ansible/blob/c2f528b7686c9573aa8493357219e0bbe4416dd3/changelogs/fragments/reboot-change-default-boot-command.yaml) as an example.